src/debasmita/A21_StringCode2.java

```diff
@@ -0,0 +1,21 @@
+package debasmita;
+
+public class A21_StringCode2 {
+     
+	  String temp="";
```

Comment:
if this variable is common for other methods or same variable require in other methods then we should write it at the instance level.